MoEngagePluginBase/MoEngagePluginUtils.swift

```diff
@@ -27,6 +27,22 @@ public class MoEngagePluginUtils {
         return appIdDict
     }
     
+        static public func fetchShouldTrackUserAttrBooleanAsNumberFromPayload(attribute: [String: Any]) -> Bool {
+        if let initConfig = attribute[MoEngagePluginConstants.General.initConfig] as? [String: Any],
+           let analyticsConfig = initConfig[MoEngagePluginConstants.General.analyticsConfig] as? [String: Any],
+           let shouldTrackUserAttributeBooleanAsNumber = analyticsConfig[MoEngagePluginConstants.General.shouldTrackUserAttributeBooleanAsNumber] {
+            return shouldTrackUserAttributeBooleanAsNumber as! Bool
```

Comment:
> @uday-engg-sdk no force unwrapping

Addressed.

Reply:
> @uday-engg-sdk remove public

If we remove public it won't be accessible in `MoECapacitorCorePlugin` class as it will have internal protection level.